packages/react-scripts/scripts/build.js

```diff
@@ -13,6 +13,13 @@
 // Do this as the first thing so that any code reading it knows the right env.
 process.env.NODE_ENV = 'production';
 
+// Makes end-to-end testing crash on unhandled rejections instead of silently
```

Comment:
Maybe replace "end-to-end testing" with "the script"?
User will see this after ejecting, and there's no reason why they should remove it.